bin/generate-sdk-package-json.js

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env node
+/*eslint-env node */
+/* eslint-disable import/no-commonjs, import/order, no-console */
+const fs = require("fs");
+const path = require("path");
+
+const IGNORED_PACKAGES = ["react", "react-dom"];
+
+function filterReactDependencies(object) {
+  const result = {};
+
+  Object.entries(object).forEach(([packageName, version]) => {
+    if (!IGNORED_PACKAGES.includes(packageName)) {
+      result[packageName] = version;
+    }
+  });
+
+  return result;
+}
+
+function generateSdkPackage() {
+  let maybeCommitHash = process.argv[2];
+
+  if (maybeCommitHash) {
+    // get short commit hash
+    maybeCommitHash = maybeCommitHash.slice(0, 7);
+  }
+
+  const mainPackageJson = fs.readFileSync(
+    path.resolve("./package.json"),
+    "utf-8",
+  );
+  const mainPackageJsonContent = JSON.parse(mainPackageJson);
+
+  const sdkPackageTemplateJson = fs.readFileSync(
+    path.resolve(
+      "./enterprise/frontend/src/embedding-sdk/package.template.json",
+    ),
+    "utf-8",
+  );
+  const sdkPackageTemplateJsonContent = JSON.parse(sdkPackageTemplateJson);
+
+  const todayDate = new Date().toJSON().slice(0, 10).replaceAll("-", "");
+
+  const mergedContent = {
+    ...sdkPackageTemplateJsonContent,
+    dependencies: filterReactDependencies(mainPackageJsonContent.dependencies),
```

Comment:
The function `filterReactDependencies` sounds more like it only filter React dependencies. However, it's more like _filtering out_ React dependencies. So I think the name
```suggestion
    dependencies: filterOutReactDependencies(mainPackageJsonContent.dependencies),
```

Or

```suggestion
    dependencies: excludeReactDependencies(mainPackageJsonContent.dependencies),
```

would be more appropriate